src/meta/src/hummock/compaction/picker/base_level_compaction_picker.rs

```diff
@@ -163,9 +163,7 @@ impl LevelCompactionPicker {
             target_level_size += sst.file_size;
         }
 
-        if target_level_size > l0_total_file_size
-            && l0_total_file_size < self.config.max_compaction_bytes
-        {
+        if target_level_size > l0_total_file_size {
```

Comment:
how about  `&& l0_compact_file_count < self.config.level0_max_compact_file_number `?

Avoiding excessive l0 growth and ensuring that other levels of compaction are not starved out by level_selector score calculation

Reply:
agree